lsbfdump/src/test/scala/org/apache/daffodil/lsbfdump/LSBFDumpTest.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.daffodil.lsbfdump
+
+import org.apache.daffodil.lsbfDump.LSBFDump
+import org.junit.Assert._
+import org.junit.Test
+
+import java.io.File
+import java.nio.file.Paths
+
+class LSBFDumpTest {
+
+  @Test
+  def testDump1(): Unit = {
```

Comment:
Note: I've not gotten these unit tests to run via `sbt test` despite the fact that the sbt-native-image stuff supposedly supports a junit runner. I had to modify the build.sbt to eliminate the native-image compilation (i.e., make it an ordinary Scala JVM project) in order to run these unit tests. 

They do run fine under IntelliJ IDEA via right-click 'run all tests'. But I think that just uses the GraalVM like an ordinary JVM.